js/.jscsrc

```diff
@@ -1,4 +1,5 @@
 {
+    "requireSemicolons": true,
```

Comment:
This was more to be consistent with what the previous style defined (the jsfmtrc required semicolons). We can move away from them, I have no strong opinions either way.